### PR TITLE
hasConflict define in Beans

### PR DESCRIPTION
**Pull request by apgsga-jhe**
hasconflict added to Beans

### CODE REVIEW

**Comment by apgsga-jhe** [comment]
@chhex : on piper side ... was it also what you thought with hasConflict at Beans level?

File: apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/MavenArtifactBean.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	private String groupId;
 	private String name;
 	private String version;
+	private boolean hasConflict = false;
```
Comment (chhex) [inline apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/MavenArtifactBean.java]: Here i would use the keyword transient

Comment (chhex) [reply apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/MavenArtifactBean.java]: So it's transient private boolean

File: apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/DbObjectBean.java [hunk 2]
```diff
@@ -11,6 +11,7 @@
 	private String fileName;
 	private String filePath;
 	private String moduleName;
+	private boolean hasConflict = false;
```
Comment (chhex) [inline apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/DbObjectBean.java]: See comment below

File: apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/MavenArtifactBean.java (hunk 1 above)
Comment (chhex) [reply apg-patch-service-common/src/main/java/com/apgsga/microservice/patch/api/impl/MavenArtifactBean.java]: And possibly you need also the @JsonIgnore for the property

**Pull request merged**